app/lib/features/share/action/share_space_object_action.dart

```diff
@@ -133,10 +135,25 @@ class ShareSpaceObjectActionUI extends ConsumerWidget {
   Widget externalShareOptionsUI(BuildContext context) {
     return ExternalShareOptions(
       qrContent: internalLink,
+      qrCodeHeader: qrCodeHeader(context),
       shareContentBuilder: shareContentBuilder,
     );
   }
 
+  Widget? qrCodeHeader(BuildContext context) {
```

Comment:
Minor feature (and useful for debugging): show the _to be shared_ information within the QRcode above the QRcode as well.